examples/conda-env-kernel-image/app-image-config-input.json

```diff
@@ -3,12 +3,12 @@
     "KernelGatewayImageConfig": {
         "KernelSpecs": [
             {
-                "Name": "conda-env-myenv-py",
+                "Name": "python3",
                 "DisplayName": "Python [conda env: myenv]"
             }
         ],
         "FileSystemConfig": {
-            "MountPath": "/root",
+            "MountPath": "/home/sagemaker-user",
             "DefaultUid": 0,
             "DefaultGid": 0
```

Comment:
https://docs.aws.amazon.com/sagemaker/latest/APIReference/API_FileSystemConfig.html

why change MounPath. It appears user inside the container is still root

Reply:
If you leave it root, than it will fail to mount.

Reply:
I tested with existing config and kerne launch succeeded without this change. can you drop this change?

Reply:
Reverted to `"MountPath": "/root",`, tested it on my Studio domain and it worked. 
Pushed changes.